spec/unit/blockscore/person_spec.rb

```diff
@@ -1,42 +1,180 @@
+            address_city: Faker::Address.city,
+            address_subdivision: Faker::Address.state_abbr,
+            address_postal_code: Faker::Address.postcode,
+            address_country_code: 'US'
+          )
+        end
+
+        it { is_expected.to be_persisted }
+        it { expect(person.class).to be BlockScore::Person }
+      end
+    end
+
+    pending '.find' do
+      context 'valid person id' do
+        let(:person_id)  { create(:person).id }
+        subject(:person) { BlockScore::Person.find(person_id) }
+
+        it { is_expected.to be_persisted }
+        it { expect(person.name_first).not_to be_empty }
+      end
+
+      context 'invalid person id' do
+        let(:person_id) { '6c89646eea50fcaa42ca1fe1667a470b' }
+
+        it { expect { BlockScore::Person.find(person_id) }.to raise_error BlockScore::RecordNotFound }
+      end
+    end
+
+    describe '.retrieve' do
+      let(:person_id)  { create(:person).id }
+      subject(:person) { BlockScore::Person.retrieve(person_id) }
+
+      it { is_expected.to be_persisted }
+      it { expect(person.name_first).not_to be_empty }
+      it { expect(person.class).to be BlockScore::Person }
+    end
 
-    context 'when person has existing question sets' do
-      let(:person) { create(:person, question_set_count: 1) }
-      subject(:question_sets) { person.question_sets }
-      its(:size) { should eql(1) }
-      its(:ids) { should be(person.attributes[:question_sets]) }
-      its(:first) { should be_an_instance_of(QuestionSet) }
+    describe '.all' do
+      let(:uniq_token_one) { '4c2c019a1e3c33442644d9f52c7a93f7' }
+      let(:uniq_token_two) { 'b19a680671c6c006c6105aa450837668' }
+      subject(:people) { BlockScore::Person.all }
 
-      it 'should load attributes' do
-        expect(question_sets.first.attributes).not_to be(nil)
+      before do
+        create(:person, name_first: 'John', name_last: uniq_token_one)
+        create(:person, name_first: 'John', name_last: uniq_token_two)
+      end
+
+      it { is_expected.not_to be_empty }
+      it { expect(people[0].name_last).to eq uniq_token_two }
+      it { expect(people[1].name_last).to eq uniq_token_one }
+    end
+
+    describe '#valid?' do
+      context 'valid person' do
+        subject(:person) { create(:valid_person) }
+
+        it { is_expected.to be_persisted }
+        it { expect(person.valid?).to be true }
+        it { expect(person.class).to be BlockScore::Person }
+      end
+
+      context 'invalid person' do
+        subject(:person) { create(:invalid_person) }
+
+        it { is_expected.to be_persisted }
+        it { expect(person.valid?).to be false }
+        it { expect(person.class).to be BlockScore::Person }
+      end
+    end
+
+    describe '#invalid?' do
+      context 'valid person' do
+        subject(:person) { create(:valid_person) }
+
+        it { is_expected.to be_persisted }
+        it { expect(person.invalid?).to be false }
```

Comment:
might as well use the predicate inflection you used in the line above for `invalid?` as well